org.mhogomchungu.media_downloader.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+
+	<id>org.mhogomchungu.media_downloader</id>
+
+	<name>Media Downloader</name>
+
+	<summary>A Qt based GUI frontend to yt-dlp, gallery-dl, lux, svtplay-dl and others</summary>
+
+	<metadata_license>FSFAP</metadata_license>
+
+	<project_license>GPL-2.0-or-later</project_license>
+
+	<url  type="homepage">https://github.com/mhogomchungu/media-downloader</url>
+
+	<content_rating type="oars-1.1" />
+
+	<developer id="org.mhogomchungu.media_downloader">
+		  <name>Francis Banyikwa</name>
+	</developer>
+
+	<description><p>This application is a front end to command line tools that deals with downloading of online media and can be used to download video files, audio files and images from many website.</p></description>
+
+	<icon type="stock">org.mhogomchungu.media_downloader</icon>
+
+	<launchable type="desktop-id">org.mhogomchungu.media_downloader.desktop</launchable>
+
+	<screenshots>
+		<screenshot type="default">
+			<image>https://raw.githubusercontent.com/mhogomchungu/media-downloader/main/images/media-downloader-1.png</image>
```

Comment:
never reference a screenshot on a branch. it's never a permanent URL.

(same below)

Reply:
How should i reference an image hosted on github? I have been doing it this was for more than a decade now and they never broke because they are on the main branch and the branch will never get deleted.

Reply:
but the file can be deleted. so in more than a decade you don't know how to give the URL of a specific revision? Just replace the branch name by a tag name OR a commit sha1. Simple as that.